config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   namespace :teachers do
     resources :exams, only: [:index]
-  end
+    resources :questions
 
+    get 'questions/show'
```

Comment:
`resources :questions` include all actions, you do not need additional get and new
to restrict some actions you can use `only: [:new, :show]`